hastexo_guacamole_client/guacamole/client.py

```diff
@@ -0,0 +1,221 @@
+"""
+The underlying work for this code is published under the MIT License (MIT)
+(https://pypi.org/project/pyguacamole/)
+"""
+
+import socket
+import logging
+
+from .exceptions import GuacamoleError
+
+from .instruction import INST_TERM
+from .instruction import GuacamoleInstruction as Instruction
+
+# supported protocols
+PROTOCOLS = ('vnc', 'rdp', 'ssh')
+
+PROTOCOL_NAME = 'guacamole'
```

Comment:
Having two variables named `PROTOCOLS` and `PROTOCOL_NAME` that have next-to-nothing to do with each other strikes me as difficult to parse intuitively. Can we rename one of them?

Reply:
Good note, `PROTOCOL_NAME` is actually not even used anywhere (at least anymore), I can just drop it. :)

Reply:
Gone in ea3cebaa11390a4b50b70df8cec8e09efc01283b.